libs/application/ui-fields/src/lib/VehicleRadioFormField/VehicleDetails.ts

```diff
@@ -0,0 +1,22 @@
+interface ValidationErrorMessage {
+  errorNo?: string | null
+  defaultMessage?: string | null
+}
+
+export interface VehicleDetails {
+  permno?: string
+  make?: string
+  color?: string
+  role?: string
+  requireMileage?: boolean | null
+  mileageReading?: string | null
+  isDebtLess?: boolean | null
+  validationErrorMessages?: ValidationErrorMessage[] | null
```

Comment:
Is the undefined that comes from "?" possibly enough? Is the "null" needed as well?